### PR TITLE
app-admin/syslog-summary: fix adjacent line ignore

### PR DESCRIPTION
**Pull request by lmiphay**
fix for not ignoring a second line when two lines which should be
ignored come together - backport fix from:

  https://github.com/dpaleino/syslog-summary/pull/1/files

switch to EAPI 7

Signed-off-by: Paul Healy <lmiphay@gmail.com>

### CODE REVIEW

File: app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~sparc ~x86"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}"
+
+PATCHES=( "${FILESDIR}/fix-ignore-code.patch" )
```
Comment (juippis) [inline app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild]: The patchfile should have an identifiable name, like `${P}-fix-ignore-code.patch`.

Rest looks good.

File: app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~sparc ~x86"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}"
```
Comment (juippis) [inline app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild]: Might also need `BDEPEND="${PYTHON_DEPS}"`.

File: app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild (hunk 1 above)
Comment (lmiphay) [reply app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild]: Yep, agree.

File: app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild (hunk 2 above)
Comment (lmiphay) [reply app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild]: This one I don't know about?

There isn't as far as I can see any python called during the package build? On that basis DEPEND and BDEPEND should probably be empty (as DEPEND is now).

However happy to have the error of my ways pointed out!

File: app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild (hunk 1 above)
Comment (lmiphay) [reply app-admin/syslog-summary/syslog-summary-1.14-r3.ebuild]: (renamed)

**Comment by lmiphay** [comment]
Thanks again for the review @juippis !

**Pull request closed without merging**